client/modules/Run/pages/RunListPage/RunListPage.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+// Import Components
+import RunList from '../../components/RunList';
+
+// Import Actions
+import { fetchRuns, deleteRunRequest } from '../../../Run/RunActions';
+
+// Import Selectors
+import { getRuns } from '../../../Run/RunReducer';
+
+class RunListPage extends Component {
+  componentDidMount() {
+    this.props.dispatch(fetchRuns());
+  }
+
+  handleDeleteRun = run => {
+    if (confirm('Do you want to delete this test run?')) { // eslint-disable-line
```

Comment:
why did eslint need to be disabled for this line?  can you make it a specific disable?

Reply:
Made it a specific disable. The linter does not like popup windows. Which, is totally fair...but the way this boilerplate came.